recipe/meta.yaml

```diff
@@ -10,11 +10,11 @@ package:
 source:
   fn: scikit-umfpack--{{ version }}.tar.gz
   url: https://files.pythonhosted.org/packages/source/s/scikit-umfpack/scikit-umfpack-{{ version }}.tar.gz
-  sha256: 9c8935717b17e8b43ad8ec989c2ca0e48c1e1b01fe0d1a16e19feecde2ee9524
+  sha256: 6236856ed34a74a541028a80989372e956ccfbd2751de0ee74b7e0800f50f3b7
 
 build:
   skip: True  # [win]
-  number: 202
+  number: 0
```

Comment:
The `200` build number is a terrible hack to start with, but if that hack is still needed the bot is not ready for it.
Ping @CJ-Wright and @justcalamari

Reply:
I did not notice this prior to merging.  I can change to 200 in a new PR assuming that is still required?